.github/workflows/ci.yaml

```diff
@@ -64,6 +64,11 @@ jobs:
       - name: Checkout the repo
         uses: actions/checkout@v3
 
+      - name: Workaround runner image issue
+        if: matrix.os == 'ubuntu-20.04'
+        # https://github.com/actions/runner-images/issues/7061
```

Comment:
Sounds good, but I would also add a TODO here that says: This is a fix for thing X and once it is fixed, pls remove this step. You can also mention estimated time of it being fixed: for example it should be fixed by the end of Feb 2023.

Why? With the comment you left, it will take very long time till somebody checks that link -> there is no reason to look at it.

But with the comment I suggested above, the next time somebody looks at this piece of code, they will likely follow up on it, especially if they see that the mentioned date has passed, so we will have this fix no longer than we should. And I think that is good, I would rather not unnecessary have it here if it is not needed.

Reply:
Makes sense, I'll add it in the next update to the workflow (when I'll be changing the Node.js version back to `18`)